src/promptflow/promptflow/_core/operation_context.py

```diff
@@ -77,9 +74,7 @@ def __getattr__(self, name):
         Returns:
             int, float, str, bool, or None: The value of the attribute.
         """
-        if name == self._USER_AGENT:
```

Comment:
may also remove self._USER_AGENT defined in class

Reply:
It has been deleted.

Reply:
![image](https://github.com/microsoft/promptflow/assets/47586720/67cc8320-9dbc-4026-80e0-92b77ecc70a7)
I mean this line

Reply:
ok, done.